Dockerfile.worker

```diff
@@ -113,6 +113,13 @@ RUN mkdir src &&\
 
 ENV CODE /code
 WORKDIR $CODE
+
+# Install dependencies
+RUN wget -O libutf8proc-dev.deb http://ftp.ch.debian.org/debian/pool/main/u/utf8proc/libutf8proc-dev_1.3.1-2_amd64.deb
+RUN wget -O libutf8proc1.deb http://ftp.ch.debian.org/debian/pool/main/u/utf8proc/libutf8proc1_1.3.1-2_amd64.deb
```

Comment:
I guess the versions of `libutf8proc-dev` and `libutf8proc` should be kept in sync. Should the version number be extracted to a variable/constant?